### DISCUSSION TITLE
[Step-by-Step and DBSCAN]:Number of double-strand breaks (DSBs) discrepancy between Step-by-Step and DBSCAN

### DISCUSSION THREAD

**Discussion by thanhtaiphys** [topas-nbio/TOPAS-nBio#33] [Simulation-question]
### Simulation question

I am currently scoring the number of double-strand breaks (DSBs) caused by Lu-177 using TOPAS-nBio. The geometry is set up as follows: 
# Cell
s:Ge/MyCell/Type                  = "TsSphericalCell"
s:Ge/MyCell/Material              = "G4_WATER"
d:Ge/MyCell/CellRadius            = **10. um**

# Nucleus
s:Ge/Nucleus/Type                       = "TsNucleus"
s:Ge/Nucleus/Parent                     = "MyCell"
s:Ge/Nucleus/Material                   = "G4_WATER"
s:Ge/Nucleus/Color                      = "White"
s:Ge/Nucleus/DNAModel	     = "Sphere"
d:Ge/Nucleus/NucleusRadius              = **4.65 um**
i:Ge/Nucleus/HilbertCurveLayer          = 4
i:Ge/Nucleus/HilbertCurve3DRepeat       = 30

# Source
s:So/RadionuclideSource/BeamParticle                         = "GenericIon(71, 177)
s:So/RadionuclideSource/Component                           = "MyCell"
ic:So/RadionuclideSource/NumberOfHistoriesInRun    = 1000            
ic:So/RadionuclideSource/NumberOfSourcePoints       = 1500 

I placed the Lu-177 source in the cell, with all other parameters kept the same. I then used both the **Step-by-Step** and **DBSCAN** methods to record the **number of DSBs**.

I expected that the number of DSBs recorded by the Step-by-Step method would be higher than that recorded by DBSCAN because Step-by-Step accounts for both direct and indirect damage. Surprisingly, the results showed that the number of DSBs scored using DBSCAN (23 DSBs) is significantly higher than the number scored by the Step-by-Step method (5 DSBs). The results I obtained were the opposite of what I expected!

I am uncertain if I have made any mistakes or if there are additional factors I need to consider, such as normalizing the DSBs from the SBS method by dose or adjusting the parameters for DBSCAN clustering

Thank you!

### TOPAS-nBio category

Scoring

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
Hi there,
Are the configuration parameters of the TsNucleus set to mimic the same DNA density that DBSCAN simulates? 
Cheers.

**Comment by thanhtaiphys** [comment]
Thank you for your response. Just to clarify, I am using the same geometric configuration in a single simulation file. The only difference is that I am using two different scorers — Step-by-Step and DBSCAN — to score the number of double-strand breaks (DSBs). I have ensured that everything else is kept the same.

**Below are the settings for the DBSCAN scorer:**

s:Sc/DBSCAN/Quantity = "DBSCAN"
s:Sc/DBSCAN/Component  = "Nucleus"
s:Sc/DBSCAN/OutputFile = "DBSCAN_Y90"
s:Sc/DBSCAN/OutputType = "ASCII"
b:Sc/DBSCAN/OutputToConsole = "True"
s:Sc/DBSCAN/IfOutputFileAlreadyExists = "Increment"
d:Sc/DBSCAN/MinimumDistanceForDSB       = 3.2 nm  # Distance betwen SSB to form a DSB ~10 bp
u:Sc/DBSCAN/SampleHitsWithProbability   = 0.16    # Hits are scorer with 16% probability
d:Sc/DBSCAN/LowerEnergyForSamplingSSB   = 5.0 eV  # Hits are sampled from a ramp distribution between
d:Sc/DBSCAN/UpperEnergyForSamplingSSB   = 37.5 eV # these two values: 5.0 eV -> 0.0; 37.5 eV -> 1.0
i:Sc/DBSCAN/MinimumNumberofSSBtoFormDSB = 2       # Minimum number of SSB to form a DSB

**For the Step-by-Step method, I am using the default scorer settings in TOPAS-nBio with adjustment: DirectDamageThreshold: 17.5 eV**

is it true that if the simulation is set up correctly, the number of DSBs scored by the Step-by-Step method should be higher than the number recorded by DBSCAN?
